docs/en/observability/quickstarts/monitor-k8s-otel.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[monitor-k8s-otel-edot]]
+= Quickstart: Unified Kubernetes Observability with Elastic Distributions of OpenTelemetry (EDOT)
+
+preview::[]
+
+In this quickstart guide, you will learn how to send Kubernetes logs, metrics, and application traces to Elasticsearch using the https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator] to orchestrate the https://github.com/elastic/opentelemetry/tree/main[Elastic Distributions of OpenTelemetry] (EDOT) Collectors and SDK instances you will need.
+
+All the components will be deployed through the https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-kube-stack[opentelemetry-kube-stack] helm chart. They include:
+
+* https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator]
+* `DaemonSet` EDOT Collector configured for node level metrics.
+* `Deployment` EDOT Collector configured for cluster level metrics.
+* Instrumentation object for applications https://opentelemetry.io/docs/kubernetes/operator/automatic/[auto-instrumentation].
+
+For a more detailed description of the components and advanced configuration refer to the https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md[elastic/opentelemetry] GitHub repository.
+
+[discrete]
+== Prerequisites
+
+* An {es} cluster, self-managed or a hosted {ess} deployment on {ess-trial}[{ecloud}]. The deployment includes an {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data.
```

Comment:
This text is slightly different from the text we use in the other quick start guides. Let's match the text we use in those guides for now. If you think it should be updated, we should do it in all three. We can open an issue to address that.

Reply:
I was assuming `self-managed` clusters could also use the onboarding UX, but honestly I haven't checked.
I'm ok about leaving the same text as in the other guides.